crates/ark/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ark"
-version = "0.1.120"
+version = "0.1.121"
```

Comment:
By convention we bump ark versions in a separate PR (that doesn't need a review), because it's possible that the ark version is bumped by someoneelse while this PR is still in review.

Reply:
Roger that.